### PR TITLE
Bugfix/photo submission

### PR DESCRIPTION
**Pull request by munroTom**
Relating to: https://trello.com/c/iwzOvodV/116-unexpected-behaviour-in-photo-submission-form

Sorry, there seem to be a lot of auto formatting errors, if it's an issue let me know and I'll try to sort it out, I had disabled the formatting extension I was using but that doesn't seem to have cut it!

### CODE REVIEW

File: src/components/PhotoPage/TitleTextField.js [hunk 1]
```diff
@@ -13,40 +13,41 @@ const styles = theme => ({
 });
 class TitleTextField extends Component {
 
-  dataFormater = value =>{
+  dataFormater = value => {
+    if (!value) return String(value)
```
Comment (munroTom) [inline src/components/PhotoPage/TitleTextField.js]: This was to stop the `number of pieces` field being an undeletable zero

File: src/components/PhotoPage/index.js [hunk 2]
```diff
@@ -217,15 +219,28 @@ class PhotoPage extends Component {
...
+      }, error => {
+        this.openDialog('Photo upload was canceled');
+      }, () => {
+        this.openDialog("Photo was uploaded successfully. It will be reviewed by our moderation team.", this.handleClosePhotoPage);
+      }
       );
     }
   }
 
+  sanitiseField = (field, key) => {
```
Comment (munroTom) [inline src/components/PhotoPage/index.js]: Sanitising brand names

File: src/components/PhotoPage/index.js [hunk 3]
```diff
@@ -175,29 +175,31 @@ class PhotoPage extends Component {
...
 
     let filteredFields = {};
-    Object.entries(fieldsJustValues).forEach(([key,value]) =>{
-       if(value){
-         filteredFields[key] = value;
-       }
-     });
+    Object.entries(fieldsJustValues).forEach(([key, value]) => {
+      if (value) {
+        value = this.sanitiseField(value, key)
```
Comment (munroTom) [inline src/components/PhotoPage/index.js]: Sanitising brand names

**Comment by globalavocado** [comment]
Thanks very much for this, we have made some small changes and merged with photos.

**Pull request closed without merging**